.taskcluster.yml

```diff
@@ -87,38 +87,67 @@ tasks:
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
       - $if: 'tasks_for == "github-push" && head_branch[:10] == "refs/tags/"'
         then:
+          taskId: {$eval: as_slugid("build_release")}
           dependencies:
             - {$eval: as_slugid("lint_task")}
             - {$eval: as_slugid("test_task")}
-          scopes:
-            - secrets:get:project/relman/dump_syms/deploy
           created: {$fromNow: ''}
           deadline: {$fromNow: '2 hour'}
           provisionerId: proj-relman
           workerType: ci
           payload:
-            features:
-              taskclusterProxy:
-                true
             maxRunTime: 3600
             image: "rust:buster"
             env:
               PROJECT_NAME: dump_syms
             command:
               - "/bin/bash"
               - "-cx"
-              - "curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py &&
-                 python3 get-pip.py &&
-                 rustup toolchain install stable &&
+              - "rustup toolchain install stable &&
                  rustup default stable &&
                  git clone --recursive --quiet ${repository} &&
-                 cd dump_syms &&
+                 cd dump_syms && mkdir -p /build &&
                  git -c advice.detachedHead=false checkout ${head_rev} &&
-                 cargo build --release &&
-                 pip3 install -r infra/requirements.txt &&
-                 python3 infra/release.py"
+                 cargo build --release --target-dir=/build &&
+                 cd /build && tar -zvcf /build.tar.gz ."
```

Comment:
This will archive a lot of files that we don't actually need, see the old release.py script or just download an archive from a previous release to see the files we need.

Reply:
yeah just dump_syms is enough